### DISCUSSION TITLE
Players change UUID after /premium and get new profile

### DISCUSSION THREAD

**Discussion by scorninpc** [games647/FastLogin#1143] [Q&A] [answered]
Some players forgot to do `/premium`, so after some progress, we remember to do `/premium`

But when player to that, all UUID change, luckperm give alert about another UUID and all progress of player are lost (by this news UUID)

Where i'm failing? should I keep auto-register enable? what can i do?

**Comment by TuxCoding** [comment, marked as answer]
No it is unrelated to auto-register. You have likely premiumUUID enabled. This has the consequence of changing the UUID when onlinemode switches.

https://github.com/games647/FastLogin/blob/2112d06162050c1b09a2b821d0763c33784a6dda/core/src/main/resources/config.yml#L85

**Reply by scorninpc** [reply]
You are very right!

I turning it false, and I'll ask to a player to test when he is online

confirmed: this is the problem. Sorry
